docs/cli/token.md

```diff
@@ -149,7 +149,7 @@ Flag | Description
 Available as of the October 2023 releases (v1.28.2+k3s1, v1.27.7+k3s1, v1.26.10+k3s1, v1.25.15+k3s1).
 :::
 
-Rotate original server token with a new bootstrap token. After running this command, all servers and any agents that originally joined with the old token must be restarted with the new token.
+Rotate original server token with a new bootstrap token. After running this command, all servers and any agents that originally joined with the old token must be restarted with the new token. This bootstrap token never expires.
```

Comment:
Ugh our reuse of the term "bootstrap" sucks. This is the server token used to encrypt the bootstrap data, it is NOT an "agent bootstrap token" and I think we should not use the term "bootstrap token" to refer to anything other than an agent bootstrap token. This should just say "server token".

Reply:
Ok, let me try to rewrite it a bit better